scripts/pre_push_hook_test.py

```diff
@@ -612,8 +676,8 @@ def mock_has_uncommitted_files():
             'Your repo is in a dirty state which prevents the linting from'
             ' working.\nStash your changes or commit them.\n' in self.print_arr)
 
-    def test_error_while_branch_change(self):
-        def mock_check_output(cmd_tokens):
+    def test_error_while_branch_change(self) -> None:
+        def mock_check_output(cmd_tokens: List[str], text: bool = True) -> str:  # pylint: disable=unused-argument
```

Comment:
Why is bool used for something called `text`? (I don't even really understand what the arg name is trying to convey, maybe some explanation would help.)

Reply:
The reason why we are using text argument is mentioned in this comment https://github.com/oppia/oppia/pull/16226#discussion_r991137346

But there we are defining it for typing purposes, here we defined text so that the mock function matches with the original function and do not throw error for missing argument.

Reply:
OK, maybe you could add a code comment + reference to explain what's going on. It reads a bit confusingly.

Reply:
As per the offline conversation, `encoding` arg is added.